### PR TITLE
#201 Have set proper run name and asset name for data quality tests

### PR DESCRIPTION
**Pull request by ajitava-git**
@sravantit25 I have set names for the run name and asset name for the tests. Could you please review the same?

### CODE REVIEW

File: tests/data_validation/great_expectations/checkpoints/comic_title_validations.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ validations:
   - batch_request:
       datasource_name: newsletter_automation_datasource
       data_connector_name: default_runtime_data_connector_name
-      data_asset_name: newsletter_automation.articles
+      data_asset_name: Comics
```
Comment (sravantit25) [inline tests/data_validation/great_expectations/checkpoints/comic_title_validations.yml:26]: @ajitava-git Were you able to test this change? I don't think data_asset_name can be changed like that. It refers to database_name.table_name. It is referred similarly in the expectation files as well.

**Review by sravantit25 — COMMENTED** [review]
Please review the data_asset name change for all the files.

File: tests/data_validation/great_expectations/checkpoints/description_validation.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ config_version: 1.0
 template_name:
 module_name: great_expectations.checkpoint
 class_name: Checkpoint
-run_name_template: 'newsletter automation-description validation'
+run_name_template: 'Ready Automation Article with Title and Description'
```
Comment (sravantit25) [inline tests/data_validation/great_expectations/checkpoints/description_validation.yml:6]: This is not applicable for Automation Article alone but all Articles. So need to name the template accordingly. Also it is for Description validation whether it is provided or not.

File: tests/data_validation/great_expectations/checkpoints/description_validation.yml [hunk 3]
```diff
@@ -23,7 +23,7 @@ validations:
   - batch_request:
       datasource_name: newsletter_automation_datasource
       data_connector_name: default_runtime_data_connector_name
-      data_asset_name: newsletter_automation.articles
+      data_asset_name: Automation corner
```
Comment (sravantit25) [inline tests/data_validation/great_expectations/checkpoints/description_validation.yml:26]: This is not for Automation corner alone but all Articles like Past week and current week too.

File: tests/data_validation/great_expectations/checkpoints/past_articles_count_validations.yml [hunk 4]
```diff
@@ -3,7 +3,7 @@ config_version: 1.0
 template_name:
 module_name: great_expectations.checkpoint
 class_name: Checkpoint
-run_name_template: 'newsletter automation-past articles validation'
+run_name_template: 'Ready Past Week Article with Title and Description'
```
Comment (sravantit25) [inline tests/data_validation/great_expectations/checkpoints/past_articles_count_validations.yml:6]: I think this checks the Past Articles count and not title and description.

**Review by sravantit25 — CHANGES_REQUESTED** [review]

File: tests/data_validation/great_expectations/checkpoints/comic_title_validations.yml (hunk 1 above)
Comment (sravantit25) [reply tests/data_validation/great_expectations/checkpoints/comic_title_validations.yml:26]: I verified and we can change the data asset name. I ran workflow against this branch and was able to get results.

File: tests/data_validation/great_expectations/checkpoints/description_validation.yml (hunk 2 above)
Comment (ajitava-git) [reply tests/data_validation/great_expectations/checkpoints/description_validation.yml]: Updated the run_name_template with All Articles Description Validation

File: tests/data_validation/great_expectations/checkpoints/description_validation.yml (hunk 3 above)
Comment (ajitava-git) [reply tests/data_validation/great_expectations/checkpoints/description_validation.yml]: Updated the data_asset_name as All Articles

File: tests/data_validation/great_expectations/checkpoints/past_articles_count_validations.yml (hunk 4 above)
Comment (ajitava-git) [reply tests/data_validation/great_expectations/checkpoints/past_articles_count_validations.yml]: Update run_name_template: 'Past Article Count'

**Review by sravantit25 — APPROVED** [review]
Verified that the changes have been addressed.

**Pull request merged**